app/templates/index.html

```diff
@@ -16,100 +16,147 @@ <h1>
-            See Digital Marketplace sales figures
-          </a>
-        </p>
-      </div>
-    </aside>
+        </li>
+
+        <li>
+          <strong>
+            <a class="govuk-link"
+               href="{{ url_for('external.info_page_for_starting_a_brief', framework_slug=dos_slug, lot_slug='user-research-participants') }}">
+              Find user research participants
+            </a>
+          </strong>
+          <p class="govuk-body">
+            eg people from a specific user group to test your service
+          </p>
+        </li>
+
+        <li>
+          <strong>
+            <a class="govuk-link"
+               href="{{ url_for('external.studios_start_page', framework_slug=dos_slug) }}">
+              Find a user research lab
+            </a>
+          </strong>
+          <p class="govuk-body">
+            eg a room to conduct research sessions
+          </p>
+        </li>
+        {% endif %}
+
+        <li>
+          <strong>
+            <a class="govuk-link"
+               href="{{ url_for('main.index_g_cloud') }}">
+              Find {{ gcloud_framework_description }}
+            </a>
+          </strong>
+          <p class="govuk-body">
+            eg content delivery networks or accounting software
+          </p>
+        </li>
+
+        <li>
+          <strong>
+            <a class="govuk-link"
+               href="{{ url_for('main.index_crown_hosting') }}">
+              Buy physical datacentre space
+            </a>
+          </strong>
+          <p class="govuk-body">
+            eg access to mission-critical datacentres
+          </p>
+        </li>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="column-one-third">
+    <nav id="app-supplier-nav" role="navigation" aria-labelledby="app-supplier-nav-heading">
+      <h2 id="app-supplier-nav-heading" class="govuk-heading-l govuk-!-font-size-27">
+        Sell services
+      </h2>
+
+      <ul class="govuk-list">
+        <li class="govuk-!-padding-bottom-6">
+          <strong>
+            <a class="govuk-link"
+               href="{{ url_for('main.list_opportunities', framework_family='digital-outcomes-and-specialists') }}">
+              View Digital Outcomes and Specialists opportunities
+            </a>
+          </strong>
+        </li>
+
+        {% if temporary_message %}
+        <li>
+          {%
+            with
+              heading = temporary_message.heading,
+              main = True,
+              subheading = temporary_message.subheading,
+              messages = temporary_message.messages,
+              message = temporary_message.message
+          %}
+            {% include "toolkit/temporary-message.html" %}
+          {% endwith %}
+        </li>
+        {% endif %}
+
+        {% if current_user.role != 'supplier' %}
+        <li class="govuk-!-padding-bottom-6">
+          <strong>
+            <a class="govuk-link"
+               {# TODO: add an external link for this #}
```

Comment:
I know this was here before - do we have a tech debt card for this?

Reply:
There is not, I can create one

Reply:
https://trello.com/c/5MAkI3PW/1237-set-up-process-for-recurrent-data-inventory-review